src/test/java/org/stellar/sdk/responses/OperationDeserializerTest.java

```diff
@@ -16,27 +16,7 @@
 import org.stellar.sdk.FormatException;
 import org.stellar.sdk.Predicate;
 import org.stellar.sdk.Price;
-import org.stellar.sdk.responses.operations.AccountMergeOperationResponse;
-import org.stellar.sdk.responses.operations.AllowTrustOperationResponse;
-import org.stellar.sdk.responses.operations.BumpSequenceOperationResponse;
-import org.stellar.sdk.responses.operations.ChangeTrustOperationResponse;
-import org.stellar.sdk.responses.operations.ClaimClaimableBalanceOperationResponse;
-import org.stellar.sdk.responses.operations.ClawbackClaimableBalanceOperationResponse;
-import org.stellar.sdk.responses.operations.ClawbackOperationResponse;
-import org.stellar.sdk.responses.operations.CreateAccountOperationResponse;
-import org.stellar.sdk.responses.operations.CreateClaimableBalanceOperationResponse;
-import org.stellar.sdk.responses.operations.EndSponsoringFutureReservesOperationResponse;
-import org.stellar.sdk.responses.operations.InflationOperationResponse;
-import org.stellar.sdk.responses.operations.LiquidityPoolDepositOperationResponse;
-import org.stellar.sdk.responses.operations.LiquidityPoolWithdrawOperationResponse;
-import org.stellar.sdk.responses.operations.ManageBuyOfferOperationResponse;
-import org.stellar.sdk.responses.operations.ManageDataOperationResponse;
-import org.stellar.sdk.responses.operations.OperationResponse;
-import org.stellar.sdk.responses.operations.PathPaymentStrictReceiveOperationResponse;
-import org.stellar.sdk.responses.operations.PathPaymentStrictSendOperationResponse;
-import org.stellar.sdk.responses.operations.PaymentOperationResponse;
-import org.stellar.sdk.responses.operations.SetOptionsOperationResponse;
-import org.stellar.sdk.responses.operations.SetTrustLineFlagsOperationResponse;
+import org.stellar.sdk.responses.operations.*;
```

Comment:
[commented earlier](https://github.com/stellar/java-stellar-sdk/pull/481/files#r1270979735), a recommendation to not use wildcard imports

Reply:
Fixed in a635bc35f5d55a0aa710ba92f01ca4961c9ea577